secret-manager/test/secretmanager.test.js

```diff
@@ -109,6 +150,13 @@ describe('Secret Manager samples', () => {
     assert.match(output, new RegExp('Created secret'));
   });
 
+  it('creates a regional secret', async () => {
```

Comment:
It's not good to chain test results like this - for example here, you create ${secretId}-2 and then in line 222 you delete it. Instead all resources for a test should be hermetic, i.e., should only be created and destroyed for that single test.

Reply:
Fixed